config/model_configs/les_isdac_box.yml

```diff
@@ -0,0 +1,32 @@
+# device: CPUSingleThreaded
+job_id: "les_isdac_box"
+initial_condition: "ISDAC" 
+subsidence: "ISDAC" 
+surface_setup: "ISDAC"
+external_forcing: "ISDAC"
+rad: "ISDAC"
+moist: "equil"
+config: "box" 
+hyperdiff: "true" 
+implicit_diffusion: false
+ode_algo: "SSP33ShuOsher"
+precip_model: "1M"
+vorticity_hyperdiffusion_coefficient: 0
+scalar_hyperdiffusion_coefficient: 0
+smagorinsky_lilly: true
+c_smag: 0.20
+x_max: 3.2e3
+y_max: 3.2e3 
+z_max: 2e3
+x_elem: 64
+y_elem: 64
+z_elem: 200
+z_stretch: false
+dt: "0.1secs" 
+t_end: "6hours" 
+dt_save_state_to_disk: "10mins"
+# restart_file: "restart/isdac/day0.0.hdf5"
+netcdf_interpolation_num_points: [64, 64, 200]
+diagnostics:
+  - short_name: [ts, ta, thetaa, ha, pfull, rhoa, ua, va, wa, hur, hus, cl, clw, cli, hussfc, evspsbl]
+    period: 5secs
```

Comment:
Let's define `reduction: average` and an appropriate `period` for this operation here. `5sec` instantaneous output might be too much here (reasonable for prototyping/ debugging I suppose).